test_suite/tests/json.rs

```diff
@@ -54,30 +44,30 @@ fn test_unit_struct() {
 	struct UnitStruct;
 
 	assert_json_for_type::<UnitStruct>(json!({
-		"id": {
-			"custom.name": 1,
-			"custom.namespace": [2],
-			"custom.params": [],
-		},
-		"def": {
-			"tuple_struct.types": []
+		"composite": {
+			"name": 1,
+			"namespace": [2],
+			"params": [],
+			"fields": [],
```

Comment:
I wonder if it makes sense to get rid of empty `params` and `fields` fields as done in `Option` before. What do you think? Maybe we should ask JS side if it makes things too complicated on their sides.

Reply:
Done this. I will test this with the `polkadot-js` PR to see how it goes.